### DISCUSSION TITLE
Mitteilungen werden nicht versendet

### DISCUSSION THREAD

**Discussion by Mate883** [brutella/hkknx-public#331] [Q&A] [answered]
Hallo zusammen,

ich habe aktuell die 3b6 installiert. Aus den Skripten funktioniert die Benachrichtigung einwandfrei, aber wenn ich z.B. bei einem Kontaktsensor über die Einstellungen eine Mitteilung aktiviere, wird keine Mitteilung versendet. Habe ich was übersehen?

**Comment by brutella** [comment]
Wie sieht die Konfiguration der Benachrichtigungen für den Kontaktsensor aus?

**Reply by Mate883** [reply]
![000490](https://github.com/user-attachments/assets/5644db23-cc6e-4809-b577-cc459b00675a)

Handy ist registriert, App ist installiert. Und ich bekomme auch Benachrichtigungen aus Skripten

**Reply by brutella** [reply]
Mit diesen Einstellungen wird eine Benachrichtigung nur versendet, wenn der Kontaktsensor vom Status "Kontakt erkannt" auf "Kein Kontakt erkannt" wechselt.

Wirf einen Blick in das [Systemprotokoll](https://hochgatterer.me/de/hkknx/docs/help/access-log/), ob bei Senden der Benachrichtigung ein Fehler dort auftaucht.

**Reply by Mate883** [reply]
Von Prinzip sollte das passen...Mitteilung bei öffnen / einschalten etc. Im Log finde ich das...habe es mit einem einfach Schalter getestet...Meldung bei Licht einschalten
![000491](https://github.com/user-attachments/assets/f158a4fb-13f7-4697-bfe5-0e6095e0b6f0)

No device...aber komischerweise bekomme ich ja bereits Notifications ??

**Reply by brutella** [reply]
Wähle testweise bei der Mitteilung für den Kontaktsensor ein bestimmtes Gerät (zB dein iPhone) aus.

**Reply by Mate883** [reply]
Jup....das habe ich gerade auch gemacht :-) Dann funktioniert es, wenn ich es aber auf "Alle Geräte" setze, nicht....

**Reply by brutella** [reply, marked as answer]
Sollte mit Version [3.0.0-b9](https://github.com/brutella/hkknx-public/releases/tag/3.0.0-b9) behoben sein.
